lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -30,6 +31,10 @@ export class LhcPeriodsOverviewModel extends Observable {
         this._pagination.observe(() => this.fetchLhcPeriods());
         this._pagination.itemsPerPageSelector$.observe(() => this.notify());
 
+        this._overviewSortModel = new SortModel();
+        this._overviewSortModel.observe(() => this._applyFilters(true));
```

Comment:
I suppose you took it from somewhere, but I think that `sortModel` should not trigger a function called `applyFilters`

Reply:
yes, refactored